.github/workflows/lts-test.yaml

```diff
@@ -0,0 +1,45 @@
+on:
+  # This should run for only pull requests for this repository's LTS branches
+  pull_request:
+    branches:
+      - '*-sp'
+name: LTS Library Test
+jobs:
+  lts-library-test:
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        # Only the downstream of this libraries are needed
+        repository:
+          - beam
+          # - google-api-java-client This does not work for LTS BOM Test due to bintray decommission
+          - google-auth-library-java
+          - google-maps-services-java
+          - grpc-java
+          - java-bigquery
+          - java-bigtable-hbase
+          - java-monitoring
+          - java-pubsub
+          - java-spanner
+          - java-storage
+          - java-trace
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-java@v1
+        with:
+          java-version: 8
+      - name: Install this repository's artifacts to local Maven repository
+        run: |
+          mvn  --batch-mode --show-version --no-transfer-progress install \
```

Comment:
Is it possible to specify some strange version number (that would not conflict) when installing this artifact?

Reply:
Does `{x-version-update:google-http-client-parent:current}` help?